services/integration.sh

```diff
@@ -74,7 +74,8 @@ function run() {
     service=$1
     colour=$2
     export LOG_LEVEL=$3
-    (cd ${DIR}/${service} && ${TOP_LEVEL}/dist/${service} -c ${service}.integration${integration_file_extension} || kill_all) \
+    instance=$4
```

Comment:
I am no bash expert but could we do something else other than waiting for an optional argument to append? Adding a 5 argument would probably confuse us quite a bit :/

Reply:
like this?: 992d69093f5a876c747e1d3cd8675a4461173ff4

Reply:
> like this?: 992d690

Yes, I think that's "better" even though maybe a bit more ugly/noisy